packages/@lwc/babel-plugin-component/src/decorators/api/transform.ts

```diff
@@ -30,19 +31,28 @@ function getPropertyBitmask(type) {
     }
 }
 
-function getSiblingGetSetPairType(propertyName, type, classBodyItems) {
+function getSiblingGetSetPairType(
+    propertyName: string,
+    type: string,
+    classBodyItems: NodePath<types.Node>[]
```

Comment:
```suggestion
    classBodyItems: types.ClassBody['body']
```

Reply:
I believe this would simplify the type casting down below.

Reply:
I actually wasn't able to get rid of the `as` casts, but I did create a helper type that tightens a types a bit:

```ts
type ClassBodyItem =
    | types.ClassMethod
    | types.ClassPrivateMethod
    | types.ClassProperty
    | types.ClassPrivateProperty
    | types.ClassAccessorProperty
    | types.StaticBlock;
```